caffe2/python/operator_test/layer_norm_op_test.py

```diff
@@ -117,6 +117,34 @@ def test_layer_norm_op(self, X, gc, dc):
             outputs_to_check=[0, 1, 2],
         )
 
+    @given(X=hu.tensors(n=1), **hu.gcs_cpu_only)
+    def test_layer_norm_op_c10(self, X, gc, dc):
+        X = X[0]
```

Comment:
Err... why would you do this. Just ask for `hu.tensor()` and get a single tensor in the first place.